webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -10,11 +10,11 @@
     graphic logo is a trademark of OpenMRS Inc.
 
 -->
-<web-app xmlns="http://java.sun.com/xml/ns/j2ee"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-app_3_0.xsd"
-	version="3.0">
-
+<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-Instance"
```

Comment:
Did you make any changes to this file?

Reply:
No, I just rebased

Reply:
So can you remove the changes that i see in this pr?

Reply:
OKay

Reply:
When I remove the changes, it causes merge conflicts. Even when I remove them locally, they still persist.

Reply:
Merge conflicts resolved